src/integrationTest/resources/application.properties

```diff
@@ -15,6 +15,8 @@ spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDri
 spring.datasource.url=jdbc:tc:postgresql:9.6://localhost/blob_router
 
 storage-blob-processing-delay-in-minutes=0
+storage-signature-algorithm: sha256withrsa
+public-key-der-file: nonprod_public_key.der
```

Comment:
should those `:` be `=` instead?

Reply:
most of the properties have `:` but a few have `=` 😕

Reply:
strange